lib/slaw/grammars/za/act_text.xsl

```diff
@@ -9,31 +9,129 @@
+  <xsl:template name="string-replace-all">
+    <xsl:param name="text" />
+    <xsl:param name="value" />
+    <xsl:param name="replacement" />
+
+    <xsl:choose>
+      <xsl:when test="$text = '' or $value = '' or not($value)">
+        <xsl:value-of select="$text" />
+      </xsl:when>
+      <xsl:when test="contains($text, $value)">
+        <xsl:value-of select="substring-before($text, $value)"/>
+        <xsl:value-of select="$replacement" />
+        <xsl:call-template name="string-replace-all">
+          <xsl:with-param name="text" select="substring-after($text, $value)" />
+          <xsl:with-param name="value" select="$value" />
+          <xsl:with-param name="replacement" select="$replacement" />
+        </xsl:call-template>
+      </xsl:when>
+      <xsl:otherwise>
+        <xsl:value-of select="$text" />
+      </xsl:otherwise>
+    </xsl:choose>
+  </xsl:template>
+
+  <!-- Escape inline markers with a backslash -->
+  <xsl:template name="escape-inlines">
+    <xsl:param name="text" />
+
+    <!-- This works from the inside out, first escaping backslash chars themselves, then escaping
+          the different types of inline markers -->
+    <xsl:call-template name="string-replace-all">
+      <xsl:with-param name="text">
+        <xsl:call-template name="string-replace-all">
+          <xsl:with-param name="text">
+            <xsl:call-template name="string-replace-all">
+              <xsl:with-param name="text">
+                <xsl:call-template name="string-replace-all">
+                  <xsl:with-param name="text">
+                    <xsl:call-template name="string-replace-all">
+                      <xsl:with-param name="text">
+                        <xsl:call-template name="string-replace-all">
+                          <xsl:with-param name="text">
+                            <xsl:call-template name="string-replace-all">
+                              <xsl:with-param name="text">
+                                <xsl:call-template name="string-replace-all">
+                                  <xsl:with-param name="text" select="$text" />
+                                  <xsl:with-param name="value"><xsl:value-of select="'\'" /></xsl:with-param>
+                                  <xsl:with-param name="replacement"><xsl:value-of select="'\\'" /></xsl:with-param>
+                                </xsl:call-template>
+                              </xsl:with-param>
+                              <xsl:with-param name="value"><xsl:value-of select="'**'" /></xsl:with-param>
+                              <xsl:with-param name="replacement"><xsl:value-of select="'\**'" /></xsl:with-param>
+                            </xsl:call-template>
+                          </xsl:with-param>
+                          <xsl:with-param name="value"><xsl:value-of select="'//'" /></xsl:with-param>
+                          <xsl:with-param name="replacement"><xsl:value-of select="'\//'" /></xsl:with-param>
```

Comment:
```suggestion
                          <xsl:with-param name="replacement"><xsl:value-of select="'\/\/'" /></xsl:with-param>
```

Reply:
It only needs the first one, because the special sequence is two forward slashes. By escaping the first, the special sequence is ignored. This is the same reason that `**` only needs `\**` and not `\*\*`. The `/` itself is not a problem (which is why `animals / minerals` doesn't need to be escaped.)